src/components/accessories/customModal/styles.scss

```diff
@@ -0,0 +1,13 @@
+.makeStyles-paper-6 {
```

Comment:
makeStyles-paper-6 is not a unique class, I mean this could change in future and lead to a unhandled behaviours.

Please, use a specific class name as for eg "custom_modal" or something like this.

Reply:
You are right I removed it